### ISSUE TITLE
Add troubleshooting guide 

### ISSUE THREAD

**Issue by Mossaka** [cpuguy83/runwasi#11]
Currently there is no troubleshotting guide. People might find it hard to follow readme to produce a hello world example. 

Known issues are, but not limited to:
1. containerd currently only support Linux. So in order to build `runwasi`, either you need to have a linux machien or run it in WSL on Windows
2. `docker buildx` is a dependency
3. `make load` is broken

**Comment by cpuguy83** [comment]
How is `make load` broken?

**Comment by cpuguy83** [comment]
containerd does run on Windows, however runwasi only supports Linux at the moment because it has to wire up networking and rootfs mounts.

**Comment by danbugs** [comment]
> How is `make load` broken?

I think @Mossaka is referring to the fact that, when trying to run it on my machine, it didn't replace the export variable `CONTAINERD_NAMESPACE` with `default` even though this line exists:
```
CONTAINERD_NAMESPACE ?= default
```

Considering `?=` only sets the value if it isn't already set, I'm guessing `export CONTAINERD_NAMESPACE` sets it to empty making `?=` not work.

**Comment by cpuguy83** [comment]
Hmmm... perhaps? I know I use this all the time.
But yeah, `export` should be moved... or even removed.

**Comment by danbugs** [comment]
If we want to keep the `export`, we could just do:
```
export CONTAINERD_NAMESPACE ?= default
```

I've tested it and it works, @cpuguy83.
